cider/tests/utils/ArrowArrayBuilderTest.cpp

```diff
@@ -288,7 +288,7 @@ TEST(ArrowArrayBuilderTest, CiderBatchConstructorTest) {
 
   CiderBatch* batch =
       new CiderBatch(schema, array, std::make_shared<CiderDefaultAllocator>());
-  EXPECT_EQ(0, batch->getBufferNum());
+  EXPECT_EQ(1, batch->getBufferNum());
```

Comment:
can you add some ut like build a struct arrowArray/CiderBatch and check whether equal?